packages/api-metadata/src/storage/fromMetadata/createFunction.ts

```diff
@@ -24,10 +31,10 @@ export interface CreateItemOptions {
  * are not known at runtime (from state_getMetadata), they need to be supplied
  * by us manually at compile time.
  */
-export default function createFunction (section: Text | string, method: Text | string, meta: StorageEntryMetadata, options: CreateItemOptions = {}): StorageEntry {
+export default function createFunction ({ meta, method, prefix, section }: CreateItemFn, options: CreateItemOptions = {}): StorageEntry {
```

Comment:
Changed the interface since 5 params is going stoopid - however, it should not be used externally, so not adding it to changelog, the prefix is already there anyway.